crates/semantic_index/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "semantic_index"
+description = "Process, chunk, and embed text as vectors for semantic search."
+version = "0.1.0"
+edition = "2021"
+publish = false
+license = "GPL-3.0-or-later"
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+path = "src/semantic_index.rs"
+
+[dependencies]
+anyhow.workspace = true
+client.workspace = true
+clock.workspace = true
+collections.workspace = true
+fs.workspace = true
+futures.workspace = true
+futures-batch = "0.6.1"
+gpui.workspace = true
+language.workspace = true
+log.workspace = true
+heed = { git = "https://github.com/meilisearch/heed", features = ["read-txn-no-tls"] }
```

Comment:
Should we pin this to a particular commit?

Could we also move this up to the Cargo workspace level?

Reply:
I was hoping by now we could pin to a version, but it seems https://github.com/meilisearch/heed/pull/241 has not been shipped. I'll pin this to a commit.